setup.py

```diff
@@ -0,0 +1,42 @@
+"""Setup for completable html XBlock."""
+
+import os
+
+from setuptools import setup
+
+
+def package_data(pkg, roots):
+    """Generic function to find package_data.
+
+    All of the files under each of the `roots` will be declared as package
+    data for package `pkg`.
+
+    """
+    data = []
+    for root in roots:
+        for dirname, _, files in os.walk(os.path.join(pkg, root)):
+            for fname in files:
+                data.append(os.path.relpath(os.path.join(dirname, fname), pkg))
+
+    return {pkg: data}
+
+
+setup(
+    name='completable-html-xblock',
+    version='0.1',
+    description='HTML XBlock will help creating and using a secure, easy-to-use and completable HTML blocks',
+    license='AGPL v3',
+    packages=[
+        'completable_html_xblock',
+    ],
+    install_requires=[
```

Comment:
Can you add [xblock-html](https://github.com/open-craft/xblock-html) to the install requirements too?
It errors out if it's not present.

Reply:
Also, can you make sure that adding `completable_html5` to a course advanced settings doesn't require adding the `html5` there too.

Reply:
> Can you add xblock-html to the install requirements too?
I actually ended pushing this to [PyPi](https://pypi.org/project/html-xblock/), because devstack uses older old pip version and I didn't want to risk potential incompatibilities between upgrades.

> Also, can you make sure that adding completable_html5 to a course advanced settings doesn't require adding the html5 there too.
It doesn't :)